Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -1224,10 +1233,14 @@ internal ItemImageInfo ItemImageInfoFromValueString(ReadOnlySpan<char> value)
                 if (nextSegment < value.Length - 1)
                 {
                     value = value[(nextSegment + 1)..];
-                    var length = value.Length;
+                    nextSegment = value.IndexOf(Delimiter);
+                    if (nextSegment == -1)
+                    {
+                        nextSegment = value.Length;
+                    }
 
-                    Span<char> blurHashSpan = stackalloc char[length];
-                    for (int i = 0; i < length; i++)
+                    Span<char> blurHashSpan = stackalloc char[nextSegment];
```

Comment:
i know its not your code but is a stackalloc really nesseary here?

Reply:
For such small fixed size byte arrays stackalloc is theoretically faster. I have no problem with its usage.